test/test-get-memory.c

```diff
@@ -26,11 +26,13 @@ TEST_IMPL(get_memory) {
   uint64_t free_mem = uv_get_free_memory();
   uint64_t total_mem = uv_get_total_memory();
   uint64_t constrained_mem = uv_get_constrained_memory();
+  uint64_t available_mem = uv_get_available_memory();
 
-  printf("free_mem=%llu, total_mem=%llu, constrained_mem=%llu\n",
+  printf("free_mem=%llu, total_mem=%llu, constrained_mem=%llu, available_memo=%llu\n",
```

Comment:
```suggestion
  printf("free_mem=%llu, total_mem=%llu, constrained_mem=%llu, available_mem=%llu\n",
```
And please wrap at 80 columns.